### ISSUE TITLE
HTTP2 and SSL certificate support

### ISSUE THREAD

**Issue by MadLittleMods** [geelen/jspm-server#4]
HTTP2/SPDY support :+1: 
- SSL keys/certificates

**Comment by geelen** [comment]
GOD yes. I have absolutely NFI how to do this, or whether it would help much in development. Do you?

**Comment by Globegitter** [comment]
@geelen yes it would help quite a lot in development as you can send all js files with one HTTP request. If you use https://github.com/molnarg/node-http2 you should be able to simply replace the `http` module as a first step and then improve on that. From my tests it seems to be backwards-compatible.
